helper/pgpkeys/encryptshares.go

```diff
@@ -0,0 +1,44 @@
+package pgpkeys
+
+import (
+	"bytes"
+	"encoding/base64"
+	"fmt"
+
+	"golang.org/x/crypto/openpgp"
+	"golang.org/x/crypto/openpgp/packet"
+)
+
+// EncryptShares takes an ordered set of Shamir key share fragments and
+// PGP public keys and encrypts each Shamir key fragment with the corresponding
+// public key
+//
+// Note: There is no corresponding test function; this functionality is
+// thoroughly tested in the init and rekey command unit tests
+func EncryptShares(secretShares *[][]byte, pgpKeys *[]string) error {
```

Comment:
This type signature is bizarre. Lets just return a new slice with the encrypted values instead of updating in place.